### PR TITLE
[Enhance] Add script for data update

### PR DESCRIPTION
**Pull request by THU17cyz**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

Motivation
After coordinate system refactor, data annotation of most datasets needs to be updated. I provide a script for fast update.

Modification
Added a script for fast data update for coordinate system refactor.

BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

Checklist
Pre-commit or other linting tools are used to fix the potential lint issues.
The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by THU17cyz** [comment]
Closed #740 and opened this pr instead because something went wrong with "files changed".

File: tools/update_data_coords.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+JOB_NAME=convert_coord_sys_data
+
+srun -p ${PARTITION} \
+    --job-name=${JOB_NAME} \
+    --gres=gpu:${GPUS_PER_NODE} \
+    --ntasks=${GPUS} \
+    --ntasks-per-node=${GPUS_PER_NODE} \
+    --kill-on-bad-exit=1 \
+    ${SRUN_ARGS} \
+    python -u tools/coord_sys_data_converter.py kitti \
```
Comment (Wuziyi616) [inline tools/update_data_coords.sh:22]: Why not setting `dataset` and those paths also args? Currently, this script can only be used to convert kitti dataset I think?

File: tools/update_data_coords.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            pkl_files.append('nuscenes_infos_train.pkl')
+        else:
+            pkl_files.append('nuscenes_infos_train_tiny.pkl')
+        update_nuscenes_or_lyft_infos(
+            root_dir=args.root_dir, out_dir=args.out_dir, pkl_files=pkl_files)
+
+        pkl_files = ['nuscenes_dbinfos_train.pkl']
+        update_outdoor_dbinfos(
+            root_dir=args.root_dir, out_dir=args.out_dir, pkl_files=pkl_files)
+    elif args.dataset == 'nuscenes' and args.version == 'v1.0-mini':
```
Comment (Wuziyi616) [inline tools/update_data_coords.py:137]: I am not familiar with this. What are the difference between `mini` and not `mini` subsets? Will we use `mini` subset in model training?

Comment (THU17cyz) [reply tools/update_data_coords.py]: I just followed `create_data.py`. Also not sure about the difference. But I fixed a related bug in the latest commit.

**Review by Wuziyi616 — APPROVED** [review]
I think the conversion is easy to understand and follow the tutorial strictly.

**Review by xiliu8006 — APPROVED** [review]

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**